x-pack/plugins/uptime/server/rest_api/pings/journeys.ts

```diff
@@ -58,16 +58,19 @@ export const createJourneyFailedStepsRoute: UMRestApiRouteFactory = (libs: UMSer
       _inspect: schema.maybe(schema.boolean()),
     }),
   },
-  handler: async ({ uptimeEsClient, request }): Promise<any> => {
+  handler: async ({ uptimeEsClient, request, response }): Promise<any> => {
     const { checkGroups } = request.query;
-    const result = await libs.requests.getJourneyFailedSteps({
-      uptimeEsClient,
-      checkGroups,
-    });
-
-    return {
-      checkGroups,
-      steps: result,
-    };
+    try {
+      const result = await libs.requests.getJourneyFailedSteps({
+        uptimeEsClient,
+        checkGroups,
+      });
+      return {
+        checkGroups,
+        steps: result,
+      };
+    } catch (e) {
+      return response.customError({ statusCode: 500, body: e });
```

Comment:
Do we have an ascribed methodology on returning error events to the client? I feel like blanket returning them is fraught as we might reveal server/ES information we don't want the client to know about.